modules/Microsoft.Synapse/privateLinkHubs/.test/common/dependencies.bicep

```diff
@@ -0,0 +1,76 @@
+@description('Optional. The location to deploy to.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the Network Security Group to create.')
+param networkSecurityGroupName string
+
+@description('Required. The name of the Virtual Network to create.')
+param virtualNetworkName string
+
+@description('Required. The name of the Private DNS Zone to create.')
```

Comment:
I'd suggest to remove this parameter. The value is not variable and hence it would be sufficient to hardcode it in the resource deployment (see comment below)

Reply:
parameter removed and hardcoded 'privatelink.azuresynapse.net'